### DISCUSSION TITLE
v1.22 feedback thread

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#44] [Ideas & Feedback]
Feedback thread for v1.22, highlighted changes:
- improved scanning speeds
- background scanning
- auto-highlighting
- blacklisting
- alternate suggestion-behavior: oversupply

**Comment by epicmeatwad** [comment]
"Not a programmer" my ass!  Youre awesome, well done again.

**Reply by noraj** [reply]
1.22.4 oversupplies is what I needed !
